### PR TITLE
fix: Fixed non-docker running of the Flask server

### PR DESCRIPTION
**Pull request by frgfm**
In #29, the need to have env variables was introduced. This PR introduces the following features:
- fixes env variables loading when not using Docker
- improves existing CI workflows

Any feedback is welcome!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,11 +2,9 @@ wheel>=0.29
 pandas>=0.24.2
 numpy>=1.18.4
 dash>=1.16.2
-dash-core-components>=1.12.1
-dash-html-components>=1.1.1
```
Comment (pechouc) [inline requirements.txt:6]: I am not sure to understand why we are erasing these requirements here?

**Review by pechouc — APPROVED** [review]
Thanks a lot @frgfm! I am not an expert at all and I don't really have enough time to dig deeper but I fully trust your input 😄

I just have a small question about requirements that are displayed as erased in the `requirements.txt` and which we are using for the web-app. Sorry if I have missed something obvious about this!

File: requirements.txt (hunk 1 above)
Comment (frgfm) [reply requirements.txt:6]: This is quite optional, it's more cosmetics than anything else: they are direct dependencies of  `dash` so installing dash installs these :)

Comment (pechouc) [reply requirements.txt:6]: Oh ok, I thought I had to install them separately when I first installed dash, nice catch!

**Pull request merged**